lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -159,9 +170,9 @@ export default function SubmitGradeForm({ disabled = false, student }) {
             'is-saved': gradeSaved,
           })}
           disabled={disabled}
-          id="lms-grade"
+          id={`lms-grade-${instanceId}`}
           ref={inputRef}
-          onKeyDown={handleKeyDown}
+          onInput={handleKeyDown}
```

Comment:
This allows the user to keyboard tab back to the validation message and have it in focus, otherwise as soon as you press tab, the message hides because onKeyDown was triggered. This more appropriately changes it to only an input change.